src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -27,7 +28,7 @@ public WinUIWebViewManager(WebView2Control nativeWebView2, IWebView2Wrapper webv
 			_contentRootDir = contentRootDir;
 		}
 
-		protected override void HandleWebResourceRequest(ICoreWebView2WebResourceRequestedEventArgsWrapper eventArgs)
+		protected override async Task HandleWebResourceRequest(ICoreWebView2WebResourceRequestedEventArgsWrapper eventArgs)
```

Comment:
Is this `awaited` anywhere?

If not, I would suggest a massive try..catch around anything that is `async` to avoid errors being lost

Reply:
Yeah it's awaited just little further down in this PR: https://github.com/dotnet/maui/pull/2055/files#diff-6f1e30516e9438af664a271d55b797b029a30f3d682b44d220803e2b1fc7da01R73